spec/requests/authorized_family_members_requests_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe "Authorized Family Members Controller", type: :request do
+  let(:partner) { create(:partner) }
+  let(:user) { create(:user, partner: partner) }
+  let!(:family) { create(:family, partner: partner) }
+
+  before do
+    sign_in(user)
+  end
+
+  describe "GET #new" do
+    it "returns http status ok" do
+      get new_authorized_family_member_path
+
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe "GET #show" do
+    let!(:authorized_family_member) { create(:authorized_family_member, family: family) }
+
+    it "returns http status ok" do
+      get authorized_family_member_path(authorized_family_member)
+
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe "POST #create" do
+    let(:authorized_family_member_params) { attributes_for(:authorized_family_member) }
+
+    it "creates and redirects to authorized_family_member_path" do
+      post authorized_family_members_path, params: {
```

Comment:
```suggestion
      post authorized_family_members_path(family_id: family.id), params: {
```
Similarly to the above comment about using the query parameter.